pangea-sdk/service/vault/api.go

```diff
@@ -324,7 +599,36 @@ func (v *vault) AsymmetricStore(ctx context.Context, input *AsymmetricStoreReque
 	return &panresp, err
 }
 
-func (v *vault) SymmetricStore(ctx context.Context, input *SymmetricStoreRequest) (*pangea.PangeaResponse[SymmetricStoreResult], error) {
+// @summary Symmetric store
+//
+// @description Import a symmetric key
+//
+// @example
+//
+//	input := &vault.SymmetricStoreRequest{
+//		Key: vault.EncodedSymmetricKey("lJkk0gCLux+Q+rPNqLPEYw=="),
+//		Algorithm: vault.SYAaes128_cfb,
+//		Purpose: vault.KPencryption,
+//		CommonStoreRequest: vault.CommonStoreRequest{
+//			Name: pangea.StringValue("my-very-secret-secret"),
+//			Folder: pangea.StringValue("/personal"),
+//			Metadata: vault.Metadata{
+//				"created_by": pangea.StringValue("John Doe"),
+//				"used_in":    pangea.StringValue("Google products"),
+//			},
+//			Tags: vault.Tags{
+//				pangea.StringValue("irs_2023"),
+//				pangea.StringValue("personal"),
+//			},
+//			RotationFrequency: pangea.StringValue("10d"),
+//			RotationState:     pangea.StringValue("deactivated"),
```

Comment:
This should be `ItemVersionState`, I think I should fix it in SDK v2. (Just comment, nothing you need to do here now)